cookbooks/ruby/attributes/versions.rb

```diff
@@ -1,5 +1,8 @@
 component = node.engineyard.environment.ruby
-ruby_version = component[:version]
+
+if node.engineyard.environment.ruby?
+  ruby_version = node.engineyard.environment.ruby[:version]
+end
```

Comment:
Please set `ruby_version` to an empty string in an else block.

Reply:
fixed